Specifications/Versioning/Versioning.csproj

```diff
@@ -1,12 +1,40 @@
 <Project Sdk="Microsoft.NET.Sdk">
-  <Import Project="../../Build/MSBuild/specs.props"/>
+  <Import Project="../../Build/MSBuild/specs.props" />
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
   </PropertyGroup>
   
   <ItemGroup>
     <ProjectReference Include="..\..\Source\Versioning\Versioning.csproj" />
   </ItemGroup>
+  
```

Comment:
This looks very funky; is it Rider putting in all kinda stuff?

Reply:
Removed it.  Not sure what that was all about.